app/src/main/java/com/example/notes/view/fragments/NoteEditFragment.kt

```diff
@@ -0,0 +1,82 @@
+package com.example.notes.view.fragments
+
+import android.os.Bundle
+import android.text.Editable
+import android.text.TextWatcher
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.Button
+import android.widget.EditText
+import androidx.fragment.app.Fragment
+import androidx.lifecycle.ViewModelProvider
+import androidx.navigation.fragment.findNavController
+import com.example.notes.R
+import com.example.notes.viewmodel.fragments.NotesEditViewModel
+
+class NoteEditFragment : Fragment() {
+    private lateinit var viewModel: NotesEditViewModel
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View {
+        return inflater.inflate(R.layout.fragment_note_edit, container, false)
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        viewModel = ViewModelProvider(
+            requireActivity(),
+            NotesEditViewModel.Factory
+        )[
+            NotesEditViewModel::class.java
+        ]
+
+        // setup text listeners
+        val titleEditText = view.findViewById<EditText>(R.id.titleEditText)
```

Comment:
Same like before, findViewById is very old way to get views.
Binding is preferred way.
why ? because when you searching views by yourself, it's prone to errors. binding do it for you automatically.